### DISCUSSION TITLE
Can we have examples reference other examples, this would cut down file size and support reusability.

### DISCUSSION THREAD

**Discussion by mcrobbj** [OAI/OpenAPI-Specification#3910] [Enhancements]
See https://github.com/OAI/OpenAPI-Specification/discussions/3902#discussioncomment-9767663

There doesn't seem to be any way around this apart from the work around suggested in the response.
Would be good if this was supported in the spec.

**Comment by handrews** [comment]
@mcrobbj we don't need both an issue and a discussion on this.  We're trying to put open-ended feature requests (where it's not clear how to solve them) in discussions while we reduce the issue backlog, so I'm going to close this but leave the discussion open.
